benchmark/cmake/modules/AddSwiftBenchmarkSuite.cmake

```diff
@@ -77,6 +77,11 @@ macro(configure_sdks_darwin)
   set(appletvos_ver "9.1")
   set(watchos_ver "2.0")
 
+  set(macosx_vendor "apple")
+  set(iphoneos_vendor "apple")
+  set(appletvos_vendor "apple")
```

Comment:
Is this used anywhere?

Reply:
Yeah, its used to calculate the target triple: https://github.com/apple/swift/blob/d86ead48d48446fb20332166760153f16b868107/benchmark/cmake/modules/AddSwiftBenchmarkSuite.cmake#L294-L319